server/itx/server.go

```diff
@@ -106,11 +108,15 @@ func (s *Server) Start(ctx context.Context) error {
 		return errors.Wrap(err, "error when starting dispatcher")
 	}
 
-	return nil
+	return s.TurnOn()
 }
 
 // Stop stops the server
 func (s *Server) Stop(ctx context.Context) error {
+	if err := s.TurnOff(); err != nil {
```

Comment:
What is the benefit if readiness is addded here?

Reply:
Readiness serves as a common mechanism to prevent Stop() being called before Start() finishes
will file a series of tasks to add Readiness into other component like Dispatcher, p2p, etc.

Reply:
after discussion, we'll determine what is the proper way to use Readiness